README.md

```diff
@@ -1,30 +1,32 @@
+# ![mag](https://raw.githubusercontent.com/nf-core/mag/master/docs/images/mag_logo.png)
+
 # nf-core/mag
-Assembly, binning and annotation of metagenomes
+
+**Assembly, binning and annotation of metagenomes**
 
 [![Build Status](https://travis-ci.org/nf-core/mag.svg?branch=master)](https://travis-ci.org/nf-core/mag)
-[![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A50.27.0-brightgreen.svg)](https://www.nextflow.io/)
+[![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A50.32.0-brightgreen.svg)](https://www.nextflow.io/)
 
 [![install with bioconda](https://img.shields.io/badge/install%20with-bioconda-brightgreen.svg)](http://bioconda.github.io/)
 [![Docker](https://img.shields.io/docker/automated/hadrieng/mag.svg)](https://hub.docker.com/r/hadrieng/mag)
-![Singularity Container available](
-https://img.shields.io/badge/singularity-available-7E4C74.svg)
+![Singularity Container available](https://img.shields.io/badge/singularity-available-7E4C74.svg)
 
 ### Introduction
-nf-core/mag: Assembly, binning and annotation of metagenomes
 
```

Comment:
Please add more description. See for example [rnaseq](https://github.com/nf-core/rnaseq), [methylseq](https://github.com/nf-core/methylseq) or the recent [rnafusion](https://github.com/nf-core/rnafusion) for examples.

Doesn't need to be long, but this will be the first point of contact for everyone using the pipeline. Should clearly say what the pipeline does: what the input files should be, what tools it runs and what it can be used for.